aspnetcore/blazor/components/virtualization.md

```diff
@@ -175,6 +175,65 @@ By default, the `Virtualize` component measures the rendering size (height) of i
 
 When making changes to items rendered by the `Virtualize` component, call <xref:Microsoft.AspNetCore.Components.ComponentBase.StateHasChanged%2A> to force re-evaluation and rerendering of the component. For more information, see <xref:blazor/components/rendering>.
 
+## Keyboard scrolling support
+
+To allow users to scroll the virtualized content using their keyboard, ensure that the virtualized elements or scroll container itself is focusable. If you do not, then keyboard scrolling does not work in Chromium-based browsers.
+
+For example, you can use a `tabindex` attribute on the scroll container:
+
+```razor
+<div style="height:500px; overflow-y:scroll" tabindex="-1">
+    <Virtualize Items="@allFlights">
+        <div class="flight-info">...</div>
+    </Virtualize>
+</div>
+```
+
+To learn more about the meaning of `tabindex` value `-1`, `0`, or other values, see [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex).
+
+## Advanced styling and scroll detection
+
+The `Virtualize` component is designed to support specific element layout mechanisms only. To understand which kinds of element layouts will work correctly, you should understand how `Virtualize` detects which elements should be visible and shows them in the correct place.
+
+If your source code looks like this:
+
+```razor
+<div style="height:500px; overflow-y:scroll" tabindex="-1">
+    <Virtualize Items="@allFlights" ItemSize="100">
+        <div class="flight-info">Flight @context.Id</div>
+    </Virtualize>
+</div>
+```
+
+... then at runtime, the `Virtualize` component will render a DOM structure similar to the following:
+
+```html
+<div style="height:500px; overflow-y:scroll" tabindex="-1">
+    <div style="height:1100px"><!-- Placeholder --></div>
+    <div class="flight-info">Flight 12</div>
+    <div class="flight-info">Flight 13</div>
+    <div class="flight-info">Flight 14</div>
+    <div class="flight-info">Flight 15</div>
+    <div class="flight-info">Flight 16</div>
+    <div style="height:3400px"><!-- Placeholder --></div>
```

Comment:
The name "Placeholder" can be confusing, because it implies that it will use Placeholder template.
I think the correct name is "Spacer", which is how it is called in source code.